sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -13,8 +13,15 @@ provider-id.setting-generator = "pluto provider-id"
 affected-services = ["kubernetes"]
 
 [metadata.settings.kubernetes.pod-infra-container-image]
-setting-generator = "schnauzer-v2 render --requires 'aws@v1' --requires 'kubernetes@v1(helpers=[pause-prefix])' --template '{{ pause-prefix settings.aws.region }}/eks/pause:3.1-eksbuild.1'"
-affected-services = ["kubernetes", "containerd"]
+affected-services = ["pod-infra-container-image"]
```

Comment:
Here we're changing a metadata list, `affected-services` - going from the two previous entries to one new one.

The closest migration is [this one](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/migration/migrations/v1.20.0/thar-be-updates-affected-services-v0-1-0/src/main.rs).